Source/CarthageKit/Errors.swift

```diff
@@ -342,7 +312,7 @@ public func < (lhs: DuplicateDependency, rhs: DuplicateDependency) -> Bool {
 		return false
 	}
 
-	for (lhsLocation, rhsLocation) in Zip2(lhs.locations, rhs.locations) {
+	for (lhsLocation, rhsLocation) in Zip2Sequence(lhs.locations, rhs.locations) {
```

Comment:
You can just call `zip` instead. The name of the sequence is an `stdlib` implementation detail